sample/ios-app/Marathonfile

```diff
@@ -17,5 +17,6 @@ vendorConfiguration:
   type: "iOS"
   derivedDataDir: "derived-data"
   sourceRoot: "sample-appUITests"
-  remoteUsername: "username"
-  remotePrivateKey: "id_rsa"
+  knownHostsPath: ${HOME}/.ssh/known_hosts
```

Comment:
Does it work with our yml parser?

Reply:
There is a commit implementing this in this PR: https://github.com/Malinskiy/marathon/pull/269/commits/32956b6d42bbb182aa8d8cee5de35c123395875f